i686 mismatched types

Getting an issue on travis, compiling for `i686-apple-darwin`

```
error[E0308]: mismatched types

   --> /Users/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/security-framework-0.1.15/src/trust.rs:124:69

    |

124 |             let certificate = SecTrustGetCertificateAtIndex(self.0, ix);

    |                                                                     ^^ expected i32, found i64
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,6 +13,12 @@ matrix:
       - osx_image: xcode7.3
         env: TARGET=x86_64-apple-ios
 
+      # 32-bit OS X 10.9
+      - osx_image: xcode6.2
+        env:
+          - FEATURES="OSX_10_9"
+          - TARGET=i686-apple-darwin
+
       # OS X 10.9
       - osx_image: xcode6.2
         env:

--- a/security-framework/src/trust.rs
+++ b/security-framework/src/trust.rs
@@ -119,7 +119,7 @@ impl SecTrust {
     /// Returns a specific certificate from the certificate chain used to evaluate trust.
     ///
     /// Note: evaluate must first be called on the SecTrust.
-    pub fn certificate_at_index(&self, ix: i64) -> Option<SecCertificate> {
+    pub fn certificate_at_index(&self, ix: CFIndex) -> Option<SecCertificate> {
         unsafe {
             let certificate = SecTrustGetCertificateAtIndex(self.0, ix);
             if certificate.is_null() {

